### ISSUE TITLE
Usage of SSH password authentication

### ISSUE THREAD

**Issue by sundhaug92** [alexjyong/TruthOrDareDiscordBot#13]
https://github.com/alexjyong/TruthOrDareDiscordBot/blob/8a747d2b39726f814e8efbc3e82c74173a626b67/.github/workflows/deployLatest.yml#LL26C38-L26C38

**Comment by alexjyong** [comment]
Yeah this was a bad choice and I should feel bad. I'll whip something up shortly.
